python/Framework.py

```diff
@@ -571,10 +527,9 @@ def configureElectronId_(self):
 
             self.process.electronMVAValueMapProducer.srcMiniAOD = self.__miniaod_electron_collection
 
+    @dep(before="configure", performs="configure")
     def configureFramework_(self):
```

Comment:
this configuration do not seem to be compulsory in the current status ?

Reply:
It prevents this method being called again (not from the constructor), because it should only be called once. We could also leave this out and make it `_configurFramework`, following the python convention that such a method is private then.

Reply:
but this is called only once when the constructor is called no ? I find the use of the decorator confusing here, because this is not something you are supposed to be playing with at analysis config level anyway ?

Reply:
Yes, changed to `_configureFramework` and removed the decorator

Reply:
thanks!